doc/classes/StreamPeerTCP.xml

```diff
@@ -61,7 +61,7 @@
 			<argument index="0" name="enabled" type="bool">
 			</argument>
 			<description>
-				Disables Nagle's algorithm to improve latency for small packets.
+				When enabled, packets will be sent immediately; otherwise, packet transfers will be delayed and combined using Nagle's Algorithm. 
 				[b]Note:[/b] For applications that send large packets or need to transfer a lot of data, this can decrease the total available bandwidth.
```

Comment:
```suggestion
				[b]Note:[/b] It's recommended to leave this disabled for applications that send large packets or need to transfer a lot of data, as enabling this can decrease the total available bandwidth.
```